CommMicro.py

```diff
@@ -139,16 +139,18 @@ def setGOVal(self,ac):
                 if cavNumA != '':
                     botPath = botPath+cavNumA[0]+"0/"+botPath+cavNumA[0]+"0_"+s1
                     caCmd = "ca://ACCL:"+liNac+":"+str(cmNumSt)+"00:RESA:"
+                    cavNums=cavNumA
 
                 if cavNumB != '':
                    botPath = botPath+cavNumB[0]+"0/"+botPath+cavNumB[0]+"0_"+s1
                    caCmd = str("ca://ACCL:"+liNac+":"+str(cmNumSt)+"00:RESB:")
+                   cavNums=cavNumA

```

Comment:
should this be cavNumB?

Reply:
yes... gosh darnit cut and paste!  I'll fix it... Thanks for catching it!